ext/fiddle/extconf.rb

```diff
@@ -46,7 +46,7 @@ def enable_debug_build_flag(flags)
 
 libffi_version = nil
 have_libffi = false
-bundle = enable_config('bundled-libffi')
+bundle = enable_config('bundled-libffi') && $mswin
```

Comment:
Can we raise an exception if an user specifies `--enable-bundled-ffi` on non mswin environment?